tests/testthat/test_model_output.R

```diff
@@ -63,6 +63,55 @@ test_that("land cover matches calibration data", {
 
 })
 
+test_that("land cover matches reference values", {
+  # Finally, test (NB rounding numeric columns to a sensible number of
+  # digits; otherwise spurious mismatches occur)
+  # Also first converts integer columns to numeric (otherwise test will
+  # fail when comparing <int> and <dbl> columns)
+  DIGITS <- 1
```

Comment:
Are we really only rounding to a single digit in this test?  That would seem to leave a lot of room for discrepancies to get missed.

Also, note that `expect_equal` (and `expect_equivalent`) can take a `tolerance` argument that gives the acceptable relative error, so `expect_equal(x,y, tolerance=1e-2)` should only flag differences greater than 1% relative error.